cluster/traefik/task_definition.tf

```diff
@@ -135,7 +138,16 @@ resource "aws_ecs_task_definition" "traefik" {
           hostPort      = var.https_port
         }
       ]
-    }
+      healthCheck = [
+        {
+          command     = ["CMD-SHELL", "healthcheck || exit 1"]
```

Comment:
There's no `healthcheck` command in Traefik. Where did this line come from?

Reply:
healthcheck    Calls Traefik /ping endpoint (disabled by default) to check the health of Traefik.

Usage: healthcheck [command] [flags] [arguments]

Use "healthcheck [command] --help" for help on any command.

Flag's usage: healthcheck [--flag=flag_argument] [-f [flag_argument]]    # set flag_argument to flag(s)
          or: healthcheck [--flag[=true|false| ]] [-f [true|false| ]]    # set true/false to boolean flag(s)